app/models/setting/puppet.rb

```diff
@@ -5,28 +5,28 @@ def self.load_defaults
-        self.set('update_environment_from_facts', N_("Foreman will update a host's environment from its facts"), false),
-        self.set('host_group_matchers_inheritance', N_("Foreman host group matchers will be inherited by children when evaluating smart class parameters"), true),
-        self.set('create_new_host_when_facts_are_uploaded', N_("Foreman will create the host when new facts are received"), true),
-        self.set('create_new_host_when_report_is_uploaded', N_("Foreman will create the host when a report is received"), true),
-        self.set('legacy_puppet_hostname', N_("Foreman will truncate hostname to 'puppet' if it starts with puppet"), false),
-        self.set('location_fact', N_("Hosts created after a puppet run will be placed in the location this fact dictates. The content of this fact should be the full label of the location."), 'foreman_location'),
-        self.set('organization_fact', N_("Hosts created after a puppet run will be placed in the organization this fact dictates. The content of this fact should be the full label of the organization."), 'foreman_organization'),
-        self.set('default_location', N_("Hosts created after a puppet run that did not send a location fact will be placed in this location"), ''),
-        self.set('default_organization', N_("Hosts created after a puppet run that did not send a organization fact will be placed in this organization"), '')
+        self.set('puppet_interval', N_("Puppet interval in minutes"), 30, N_('Puppet interval')),
+        self.set('outofsync_interval', N_("Duration in minutes after the Puppet interval for servers to be classed as out of sync."), 5, N_('Out of sync interval')),
+        self.set('default_puppet_environment', N_("Foreman will default to this puppet environment if it cannot auto detect one"), "production", N_('Default Puppet environment')),
+        self.set('modulepath',N_("Foreman will set this as the default Puppet module path if it cannot auto detect one"), "/etc/puppet/modules", N_('Module path')),
+        self.set('document_root', N_("Document root where puppetdoc files should be created"), "#{Rails.root}/public/puppet/rdoc", N_('Document root')),
+        self.set('puppetrun', N_("Enable puppetrun support"), false, N_('Puppetrun')),
+        self.set('puppet_server', N_("Default Puppet server hostname"), "puppet", N_('Puppet server')),
+        self.set('Default_variables_Lookup_Path', N_("Foreman will evaluate host smart variables in this order by default"), ["fqdn", "hostgroup", "os", "domain"], N_('Default variables lookup path')),
+        self.set('Enable_Smart_Variables_in_ENC', N_("Foreman smart variables will be exposed via the ENC yaml output"), true, N_('Enable smart variables in ENC')),
+        self.set('Parametrized_Classes_in_ENC', N_("Foreman will use the new (2.6.5+) format for classes in the ENC yaml output"), true, N_('Parametrized classes in ENC')),
```

Comment:
"Parameterized" (the setting name is misspelt, but fixing the full name is helpful!)